viam-orb-slam3_test.go

```diff
@@ -165,23 +140,6 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 		cam := &inject.Camera{}
 		switch sensor {
 		case "good_lidar":
-			cam.NextPointCloudFunc = func(ctx context.Context) (pointcloud.PointCloud, error) {
-				return pointcloud.New(), nil
-			}
-			cam.StreamFunc = func(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error) {
-				return nil, errors.New("lidar not camera")
-			}
-			cam.ProjectorFunc = func(ctx context.Context) (transform.Projector, error) {
-				return nil, transform.NewNoIntrinsicsError("")
-			}
-			cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
-				return camera.Properties{}, nil
-			}
```

Comment:
Mind explaining why we don't need to set cam.NextPointCloudFun and cam.PropertiesFunc anymore?

Reply:
See https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1138825390. TLDR, good_lidar has just become a stub to prove that we error on invalid sensor types so we dont need any of that.